workflow-templates/im-test-k6-ci.yml

```diff
@@ -0,0 +1,24 @@
+name: K6 Automated Load Tester
+
+on: 
+  push:
+    branches: 
+      - master
+      - main
```

Comment:
During the migration we'll be changing master to main, so you can change this.

Reply:
Updated